src/docs/Components/Layers/index.mdx

```diff
@@ -64,6 +64,18 @@ Layers must be used with an overlay to focus user's attention on the content.
 
 Layers are always displayed from the side of the page and appear above the content of the page with a dark overlay.
 
+## Specific variation
+
+### Extended width
+
+In the case you have complex elements to display inside the layer and need some air, you can use an extended width for the layers from the breakpoint `M`.
+
+<Highlight theme="warning" title="Be aware">
+  Never use an extended layer instead of a page.
```

Comment:
I don't really understand this sentence. Perhaps it can be reworded to make it more explicit?

Reply:
@tiloyi I will try 🙂